frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -163,9 +163,14 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                             {!featureFlag.is_remote_configuration && (
                                 <>
                                     {/* TODO: In a follow up, clean up super_groups and combine into regular ReleaseConditions component */}
-                                    {featureFlag.filters.super_groups && (
-                                        <FeatureFlagReleaseConditions readOnly isSuper filters={featureFlag.filters} />
-                                    )}
+                                    {featureFlag.filters.super_groups &&
+                                        featureFlag.filters.super_groups.length > 0 && (
+                                            <FeatureFlagReleaseConditions
+                                                readOnly
+                                                isSuper
+                                                filters={featureFlag.filters}
+                                            />
+                                        )}
```

Comment:
I didn't see the commit history; any idea how long it'd been like this?  I wonder why nobody brought it up lol

Reply:
oh interesting, so it sounds more like our API for the early access feature creation may be somewhat unprincipled

Reply:
(not a thing we need to solve here though)

Reply:
Ok, this issue was introduced 2 weeks ago by me:

https://github.com/PostHog/posthog/commit/8563391019695ada6312c83446dc45585a652c18

I should improve that logic so that if super_groups is undefined, we don't replace it with an empty one.